ngraph/core/include/ngraph/op/round.hpp

```diff
@@ -16,43 +16,49 @@
 
 #pragma once
 
-#include "ngraph/op/util/unary_elementwise_arithmetic.hpp"
+#include "ngraph/node.hpp"
+#include "ngraph/op/op.hpp"
 
 namespace ngraph
 {
     namespace op
     {
-        namespace v0
```

Comment:
Do not remove deprecated operation. Just add a new one

Reply:
Done,  now we have deprecated Round operation and a new one with two mods.